src/GitVersion.Core/VersionCalculation/Cache/GitVersionCacheKey.cs

```diff
@@ -1,15 +1,13 @@
 using GitVersion.Extensions;
 
-namespace GitVersion.Cache;
+namespace GitVersion.VersionCalculation.Cache;
```

Comment:
Really nice talk. I like it very much how you explain the SOLID pattern and moving at the end to a domain driven design. :)

So that means we need to name the Configuration namespace to Configuring!?

Reply:
Thanks! Haha! While I think using gerunds is a nice general rule, the word "configuring" doesn't work that well, imho. 😄